tools/release/lp_build_monitor_recipe.py

```diff
@@ -63,7 +94,38 @@ def get_new_builds(build_recipe, started_datetime):
     )
 
 
-def monitor_retry_builds(build_recipe, start_date):
+def monitor_retry_binary_builds(source_recipe, start_time) -> list["LPBuild"]:
```

Comment:
incomplete typing info, also the referenced type is not defined... I know that this "LPBuild" is a super dynamic type, but at least lplib offers some proxy you could use I think

Reply:
I have created classes with the appropriate inheritance that give a little bit of an idea of what is going on here. Launchpadlib sadly has 0 types locally, they are all generated by lazr restful, so there is no way (to the best of my knowledge) to give more informative type hints sadly